src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -507,7 +507,12 @@ public PublishPackageViewModel( DynamoViewModel dynamoViewModel ) : this()
 
         private void ThisPropertyChanged(object sender, PropertyChangedEventArgs e)
         {
-            if (e.PropertyName == "PackageContents") CanSubmit();
+            if (e.PropertyName == "PackageContents")
+            {
+                CanSubmit();
+                BeginInvoke(
```

Comment:
Why is BeginInvoke required here? Why can we not call RaiseCanExecuteChanged directly?